src/bridge/mod.rs

```diff
@@ -48,10 +48,14 @@ async fn start_neovim_runtime() {
     let nvim = Arc::new(session.neovim);
 
     // Check the neovim version to ensure its high enough
-    match nvim.command_output("echo has('nvim-0.4')").await.as_deref() {
+    match nvim
+        .command_output("echo has('nvim-0.9.2')")
+        .await
+        .as_deref()
+    {
         Ok("1") => {} // This is just a guard
         _ => {
-            error!("Neovide requires nvim version 0.4 or higher. Download the latest version here https://github.com/neovim/neovim/wiki/Installing-Neovim");
+            error!("Neovide requires nvim version 0.9.2 or higher. Download the latest version here https://github.com/neovim/neovim/wiki/Installing-Neovim");
```

Comment:
One benefit of doing the above suggestion is that it could be interpolated here too which would ensure they stay in sync